### PR TITLE
Add test file with no magic comment

### PR DESCRIPTION
**Pull request by joe-sharp**

### CODE REVIEW

**Comment by joe-sharp** [comment]
Plain Text:
test

HTML:
<p>test</p>

**Comment by joe-sharp** [comment]
How does this <h1>look</h1>?

**Review by kaycoinsUSA — APPROVED** [review]

**Review by kaycoinsUSA — APPROVED** [review]

**Review by Gifthub11 — COMMENTED** [review]
https://github.com/cli/cli/issues/28#issue-510192415

File: .github/workflows/mega-linter.yml [hunk 1]
```diff
@@ -45,7 +45,7 @@ jobs:
           # https://nvuillam.github.io/mega-linter/configuration/
           VALIDATE_ALL_CODEBASE: ${{ github.event_name == 'push' && github.ref == 'refs/heads/main' }} # Validates all source when push on main, else just the git diff with main. Override with true if you always want to lint all sources
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          MEGALINTER_CONFIG: https://raw.githubusercontent.com/joe-sharp/linter-configs/main/.mega-linter.yml
+          MEGALINTER_CONFIG: https://raw.githubusercontent.com/joe-sharp/linter-configs/test_rubocop/.mega-linter.yml
```
Comment (Gifthub11) [inline .github/workflows/mega-linter.yml:48]: Main

**Review by Gifthub11 — COMMENTED** [review]
https://github.com/cli/cli/issues/28#issue-510192415

File: test.rb [hunk 2]
```diff
@@ -0,0 +1 @@
+p 'hello world!'
```
Comment (Gifthub11) [inline test.rb:1]: I’m GitHub I’m fast safe and secure on internet connection and also an icon for browser

**Pull request closed without merging**